go.mod

```diff
@@ -233,7 +233,7 @@ require (
 	github.com/uudashr/gocognit v1.0.6 // indirect
 	github.com/yagipy/maintidx v1.0.0 // indirect
 	github.com/yeya24/promlinter v0.2.0 // indirect
-	github.com/zondax/hid v0.9.1 // indirect
+	github.com/zondax/hid v0.9.0 // indirect
```

Comment:
```suggestion
	github.com/zondax/hid v0.9.1 // indirect
```

Reply:
Sure, let me revert when rebasing and merging https://github.com/crescent-network/crescent/pull/159 PR